src/Twig/ReturnCmsBlockExtension.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace App\Twig;
+
+use BitBag\SyliusCmsPlugin\Entity\BlockInterface;
+use BitBag\SyliusCmsPlugin\Resolver\BlockResourceResolverInterface;
+use Twig\Extension\AbstractExtension;
+use Twig\TwigFunction;
+
+final class ReturnCmsBlockExtension extends AbstractExtension
```

Comment:
If we will manage to use the custom block template, then this extension will not be needed